besu/src/main/java/org/hyperledger/besu/cli/options/stable/TransactionPoolOptions.java

```diff
@@ -42,6 +42,7 @@ public class TransactionPoolOptions implements CLIOptions<TransactionPoolConfigu
   private static final String TX_POOL_ENABLE_SAVE_RESTORE = "--tx-pool-enable-save-restore";
   private static final String TX_POOL_SAVE_FILE = "--tx-pool-save-file";
   private static final String TX_POOL_PRICE_BUMP = "--tx-pool-price-bump";
+  private static final String TX_POOL_DISABLE_SENDER_GROUPING = "--tx-pool-disable-sender-grouping";
```

Comment:
I think it is better to make this an experimental options for the moment, just prefix it with an `X`

Reply:
I guess I should move it to https://github.com/hyperledger/besu/blob/ee7a66fc785a8a504d1e501e355492e89f6e73a3/besu/src/main/java/org/hyperledger/besu/cli/options/unstable/TransactionPoolOptions.java as well?

Reply:
correct